src/bics_bot/cogs/commands/intro_cmd.py

```diff
@@ -85,6 +94,30 @@ async def intro(
                 ephemeral=True,
             )
             return
+        
+        # Check if entered birthday is valid
```

Comment:
This block is repeated in birthday_cmd and in this file. Put this in a function.